tests/requirements.txt

```diff
@@ -16,6 +17,7 @@ pytest
 pytest-cov
 pytest-httpserver
 pytest-timeout
+rasterio>=1.3.2
```

Comment:
Might need to pin xarray too as most recent release is buggy.

Core has been updated to have   'xarray>=0.9,!=2022.6.0' but that's not released yet, so a build off this requirements file will most likely result in installing the buggy xarray-22.6.0

Reply:
`xarray` is pinned in both the `setup.cfg` and the docker image that the integration test used. This file is to "patch" the older production docker image for a regression test.